components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -848,6 +848,15 @@ - (void)viewWillTransitionToSize:(CGSize)size
   _addedContentHeight = NSNotFound;
 }
 
+- (void)setTrackingScrollView:(UIScrollView *)trackingScrollView {
```

Comment:
This will still have issues with a scroll view of different size, but that is being addressed in #8224. For future reference, once that's merged we can change this to be like so:

```
- (void)setTrackingScrollView:(UIScrollView *)trackingScrollView {
  _trackingScrollView = trackingScrollView;
  [self resetCachedState];
}
```